lang/csharp/src/apache/main/Schema/Field.cs

```diff
@@ -105,6 +105,29 @@ public enum SortOrder
         /// </summary>
         internal static JTokenEqualityComparer JtokenEqual = new JTokenEqualityComparer();
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Field"/> class.
+        /// </summary>
+        /// <param name="schema">schema for the field type.</param>
+        /// <param name="name">name of the field.</param>
+        /// <param name="aliases">list of aliases for the name of the field.</param>
+        /// <param name="pos">position of the field.</param>
+        /// <param name="doc">documentation for the field.</param>
+        /// <param name="defaultValue">field's default value if it exists.</param>
+        /// <param name="sortorder">sort order of the field.</param>
+        /// <param name="customProperties">dictionary that provides access to custom properties.</param>
+        public Field(Schema schema,
```

Comment:
Is there a reason you didn't modify the existing constructor?

Reply:
I wanted to add default values to non mandatory parameters. I had to change their order (switch between aliases and pos), and didn't want to make a breaking change.